ui/src/notebooks/context/app.tsx

```diff
@@ -19,11 +19,15 @@ type Modifier = typeof setTimeZone
 export interface AppSettingContextType {
   timeZone: TimeZone
   onSetTimeZone: Modifier
+  miniMapVisibility: boolean
+  onToggleMiniMap: () => void
 }
 
 export const DEFAULT_CONTEXT: AppSettingContextType = {
   timeZone: 'Local' as TimeZone,
   onSetTimeZone: ((() => {}) as any) as Modifier,
+  miniMapVisibility: true,
+  onToggleMiniMap: () => {},
```

Comment:
can we change this to a plain set? i can imagine that we're gonna want to change this programmatically and calling `onSetMiniMap(!miniMapVisibility)` for this case is easier than `miniMapVisibility===indendedValue?noop():onToggleMiniMap()` for the other cases